python_modules/dagster/dagster/_core/asset_graph_view/asset_graph_view.py

```diff
@@ -387,60 +381,46 @@ def compute_intersection_with_partition_keys(
             ),
         )
 
-    def create_from_time_window(self, asset_key: AssetKey, time_window: TimeWindow) -> AssetSlice:
-        return _slice_from_subset(
-            self,
-            AssetSubset(
-                asset_key=asset_key,
-                value=TimeWindowPartitionsSubset(
-                    partitions_def=_required_tw_partitions_def(self._get_partitions_def(asset_key)),
-                    num_partitions=None,
-                    included_time_windows=[time_window],
-                ),
-            ),
-        )
-
-    def create_latest_time_window_slice(self, asset_key: AssetKey) -> AssetSlice:
-        """If the underlying asset is time-window partitioned, this will return the latest complete
-        time window relative to the effective date. For example if it is daily partitioned starting
-        at midnight every day.  If the effective date is before the start of the partition definition, this will
-        return the empty time window (where both start and end are datetime.max).
-
-        If the underlying asset is unpartitioned or static partitioned and it is not empty,
-        this will return a time window from the beginning of time to the effective date. If
-        it is empty it will return the empty time window.
-
-        TODO: add language for multi-dimensional partitioning when we support it
-        TODO: add language for dynamic partitioning when we support it
+    def compute_latest_time_window_slice(
```

Comment:
`build_last_update_freshness_checks` uses the suffix `_delta` rather than `_timedelta`. I think we should be consistent. So that means changing this to `lookback_delta` or changing `build_last_update_freshness_checks`. @dpeng817 have opinons?

Default decision is to stick with what we have shipped already.

Reply:
I have no particular attachment to `_timedelta`, perfectly happy to go with `_delta`

Reply:
Yea imo prefer to use the less verbose name.